### DISCUSSION TITLE
【求助】怎样让绳网加载的更快

### DISCUSSION THREAD

**Discussion by liuying5201314** [share121/inter-knot#2403] [其他]
绳网的图片加载速度实在是太慢了，所以想要寻求一个能使绳网加载更快的方法，不知各位绳匠有何办法，求教<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="undefined" width="75" >
求大佬救救<img src="https://img-static.mihoyo.com/communityweb/upload/baaaf420abb173dac46a466b5115e3ce.gif" alt="undefined" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="undefined" width="75" >

**Comment by 6078HITHEDAY** [comment]
# ｇiｔｈｕｂ加速器
* 【『教程』手把手教你流畅访问Github-哔哩哔哩】 https://b23.tv/HkApnDP

**Comment by Qweulin** [comment]
免费的加速器终究是有限的，去买机场呗<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >
